.changeset/swift-cherries-repeat.md

```diff
@@ -0,0 +1,5 @@
+---
+'@platejs/core': minor
```

Comment:
Should be `patch`: changesets have a weird issue that release a minor change as a major release.